pandas/core/arrays/base.py

```diff
@@ -504,14 +504,22 @@ def to_numpy(
     @property
     def dtype(self) -> ExtensionDtype:
         """
-        An instance of 'ExtensionDtype'.
+        An instance of ExtensionDtype.
+
+        Examples
+        --------
+        >>> pd.array([1, 2, 3]).dtype
+        Int64Dtype()
         """
         raise AbstractMethodError(self)
 
     @property
     def shape(self) -> Shape:
         """
         Return a tuple of the array dimensions.
+
+        Since Extension Arrays are only allowed to be 1-dimensional,
+        the shape would be ``(len(self),)``.
```

Comment:
this isn't always true, can you remove it please?

Reply:
Oh thanks for pointing out. I forgot to remove this when cleaning up.